net/libdnsres/Portfile

```diff
@@ -1,39 +1,54 @@
-PortSystem        1.0
-
-name              libdnsres
-version           0.1a
-revision          3
-categories        net
-maintainers       nomaintainer
-description       A non-blocking DNS resolver library
-long_description  A non-blocking thread-safe API for resolving DNS names
-
-homepage          https://www.monkey.org/~provos/libdnsres/
-master_sites      https://www.monkey.org/~provos/
-platforms         darwin
-checksums         md5 069cc39d7ddf85c9be9ada679d5363c2
-
-patchfiles	patch-res_comp.c \
-		patch-res_debug.c \
-		patch-res_mkquery.c \
-		patch-res_init.c \
-		patch-res_query.c \
-		patch-res_send.c \
-		patch-gethostnamadr.c \
-		patch-getaddrinfo.c \
-		patch-test-simple_test.c
-
-depends_lib     port:libevent
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                libdnsres
+version             0.1a
+revision            3
+categories          net
+platforms           {darwin < 20}
+maintainers         nomaintainer
+description         A non-blocking DNS resolver library
+long_description    A non-blocking thread-safe API for resolving DNS names.
+
+homepage            https://www.monkey.org/~provos/libdnsres/
+master_sites        https://www.monkey.org/~provos/
+checksums           rmd160  05a92882d1fc0394792e9f28be4584650b46e323 \
+                    sha256  c0b94f27cb537eed42bc32ca9771f0955f717c66d34d1361e0a75600c6589307 \
+                    size    360583
+
+supported_archs     i386 ppc ppc64 x86_64
+
+patchfiles          patch-res_comp.c \
+                    patch-res_debug.c \
+                    patch-res_mkquery.c \
+                    patch-res_init.c \
+                    patch-res_query.c \
+                    patch-res_send.c \
+                    patch-gethostnamadr.c \
+                    patch-getaddrinfo.c \
+                    patch-test-simple_test.c
+
+depends_lib         port:libevent
 
 platform darwin powerpc {
-        configure.args-append   --build=powerpc-apple-darwin
+    configure.args-append \
+        --build=powerpc-apple-darwin
 }
 
-platform darwin intel {
-        configure.args-append   --build=intel-unknown-darwin
+platform darwin i386 {
+    configure.args-append \
+        --build=intel-unknown-darwin
```

Comment:
I question whether this is needed, since it has been the way it was ever since the port was added 19 years ago so it has obviously never been used since `intel` was never a valid value of `os.arch`.